.travis.yml

```diff
@@ -191,21 +191,20 @@ before_install:
           && ALPAKA_CUDA_VER_MINOR=${ALPAKA_CUDA_VER:2:1}
           && echo ${ALPAKA_CUDA_VER_MINOR}
           && ALPAKA_CUDA_CACHE_DIR=${HOME}/cache/CUDA/CUDA-${ALPAKA_CUDA_VER}
-          && if [ ${ALPAKA_CUDA_VER_MAJOR} == 7 ]
+          && if (( (( ${ALPAKA_CUDA_VER_MAJOR} == 7 )) && (( ${ALPAKA_CUDA_VER_MINOR} == 0 )) ))
```

Comment:
`if [ ${ALPAKA_CUDA_VER_MAJOR} -eq 7 ] && [ ${ALPAKA_CUDA_VER_MINOR} -eq 0 ]` is the version wish is also `sh` compatible.
But I am fine with `(( ... ))`, I was only confused because I have never seen this keyword before.

More information: http://mywiki.wooledge.org/BashFAQ/031